client/src/app/members/list/members-list.component.ts

```diff
@@ -32,7 +33,7 @@ export class MembersListComponent implements OnInit {
   ngOnInit(): void {
     this.update();
     if (!this.ngxPermissionsService.getPermissions()['member:write']) {
-      this.displayedColumns = ['lastName', 'firstName', 'school'];
+      this.displayedColumns = ['lastName', 'firstName', 'lastActive', 'school'];
     }
     this.media.subscribe((change: MediaChange) => {
       if (change.mqAlias === 'xs' && this.displayedColumns.includes('school')) {
```

Comment:
Do we want that `lastActive` column is displayed on mobile device ? I don't think so !

Reply:
Ok, for now I would like to let it here, because its the only way to see if a member is active or not. We could remove it the day we do #285

Reply:
So on mobile device, there is an issue :)
![image](https://user-images.githubusercontent.com/29222996/44082184-d0e4d1de-9fb0-11e8-8ed7-f13a2a360bea.png)

Reply:
Damn, do you have an idea to fix this?

Reply:
Yes :) I'm going to commit a fix

Reply:
![image](https://user-images.githubusercontent.com/29222996/44083101-5e3d6be8-9fb3-11e8-83e0-ae366da5c1b9.png)